pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  autoPatchelfHook,
+  libgcc,
+  xorriso,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "proxmox-auto-install-assistant";
+  version = "8.2.6";
+
+  src = fetchurl {
+    url = "http://download.proxmox.com/debian/pve/dists/bookworm/pve-no-subscription/binary-amd64/proxmox-auto-install-assistant_${version}_amd64.deb";
+    sha256 = "0z222x2pvfs0wz3gcqp5gw8gvrp9jmvkgrydn6rja6xy4rh2a9w4";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook makeWrapper ];
+  buildInputs = [
+    dpkg
+    libgcc
+    xorriso
+  ];
+
+  unpackPhase = ''
```

Comment:
Not needed? It should unpack automatically (in current dir), and the remaining work should be done in `installPhase`